tests/collection/collection-faceting.spec.ts

```diff
@@ -1,70 +1,83 @@
-import { test } from '../../tests/fixtures';
+import { test, expect } from '../../tests/fixtures';
 
-import { CollectionFacetGroupHeaderNames, FacetGroup, LayoutViewModeLocator } from '../../tests/models';
+import {
+  CollectionFacetGroupHeaderNames,
+  FacetGroup,
+  LayoutViewModeLocator
+} from '../../tests/models';
 
 test(`Verify if facets appear on first load`, async ({ collectionPage }) => {
   await test.step('Assert facet group headers count', async () => {
-    await collectionPage.collectionFacets.assertFacetGroupCount(
-      'collection', CollectionFacetGroupHeaderNames,
-    );
+    for (const header of CollectionFacetGroupHeaderNames) {
+      const facet = collectionPage.collectionFacets.facets.getByRole(
+        'heading', { name: header }
+      );
+      const facetText = (await facet.innerText()).replace(/\n/g, ' ');
+      await expect(facet).toBeVisible();
+      expect(facetText).toContain(header);
+    }
   });
 });
 
 test(`Select a facet for videos and clear facet filters`, async ({
   collectionPage,
 }) => {
+  const { collectionFacets, infiniteScroller } = collectionPage;
   await test.step(`Select "movies" from inside "Media Type" facet group and check 5 item results for "Movie" tile icon titles`, async () => {
-    await collectionPage.collectionFacets.toggleFacetSelection(
-      FacetGroup.MEDIATYPE, 'movies', 'positive',
-    );
-    await collectionPage.infiniteScroller.validateIncludedFacetedResults(
+    await collectionFacets.toggleFacetSelection(FacetGroup.MEDIATYPE, 'movies', 'positive');
+    const isFacettedCorrectly = await infiniteScroller.validateIncludedFacetedResults(
       'tile-collection-icon-title', ['Movie'], true, 5,
     );
+    expect(isFacettedCorrectly).toBeTruthy();
```

Comment:
SUGGESTION: I believe the correct spelling is "Faceted" which should be used here and throughout this PR.